pagopa/io-app
# fix: [IABT-1397] Add custom error template for `PAA_PAGAMENTO_SCONOSCIUTO`

## Short description
This PR add a custom template for payment error `PAA_PAGAMENTO_SCONOSCIUTO`

<img src="https://user-images.githubusercontent.com/6160324/210751216-cda45c05-096b-4908-9753-27af8f8a0bfd.png" width=200 />


## List of changes proposed in this pull request
- Added error mappings to `ts\screens\wallet\payment\TransactionErrorScreen.tsx`

## How to test
`yarn test` or try to pay a notice that has not been issued by a Credit Institution.
